file/1.5/modules/ROOT/pages/file-acb-configuration.adoc

```diff
@@ -0,0 +1,246 @@
+
+. Click the *Move* operation on the canvas UI to configure the following attributes in the *General* tab:
++
+include::connectors::partial$acb-config-file-connector-partial.adoc[tag="general-tab-attributes"]
+. Click the *Advanced* tab to configure the following attributes:
++
+include::connectors::partial$acb-config-file-connector-partial.adoc[tag="advanced-tab-attributes"]
+
+=== Configuring the Read Operation
+
+The *Read* operation reads the contents and metadata of a file at the specified path and returns a message whose payload is an input stream containing the file's contents. The metadata is represented as a `LocalFileAttributes` object that's expressed as message attributes (`Message#getAttributes()`).
+
+The file's MIME type is based on the file's extension. You can force the output encoding and MIME type by using the optional *Output MIME type* and *Output encoding* parameters.
+
+. Click the *Read* operation on the canvas UI to configure the following attributes in the *General* tab:
++
+[%header%autowidth.spread]
+|===
+| Attribute | Description | Required
+| File Path |The path to the file to read. | Yes
+|===
+. Click the *Advanced* tab to configure the following attributes:
++
+[%header%autowidth.spread]
+|===
+| Attribute | Description | Required
+|Lock | If selected, places a file system-level lock on the file until the input stream that the operation returns is closed or fully consumed. The lock is provided by the host file system, so its behavior might change depending on the mounted drive and the operation system on which Mule runtime is running. | No
+|Time between size check | Wait time, in milliseconds, between size checks to determine if a file is ready to read. This allows a file write to complete before processing. Mule runtime performs two size checks and waits the specified amount of time between calls. If both checks return the same value, the file is ready to read. | No
+|Time between size check unit | Time unit for *Time between size check*.| No
+| Output MIME type |MIME type of the payload that this operation outputs. | No
+| Parameters |One or more key-value pairs to append to the value of the *Output MIME type* attribute. Multiple key-value pairs are allowed. These key-value pairs enable you to store data in a variable so that any component in the flow can use it.  | No
+|Output encoding |Encoding of the payload that this component outputs. | No
+|Streaming Strategy | Configure the way data streams are consumed in Mule apps. This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See <<configuring-streaming-strategy>>. | No 
+|Target Variable | Name of the variable on which the operation's output is placed. | No
+|Target Value | An expression that's evaluated against the operation's output. The outcome of this expression is stored in the target variable. | No
```

Comment:
I may have missed this, but it would be good to remove "An" for consistency with other descriptions.